CoreWebServer/src/au/gov/asd/tac/constellation/webserver/api/SwaggerServlet.java

```diff
@@ -122,13 +122,24 @@ protected void doGet(final HttpServletRequest request, final HttpServletResponse
                     final ArrayNode params = httpMethod.putArray("parameters");
                     rs.createParameters().getParameters().entrySet().forEach(entry -> {
                         final PluginParameter<?> pp = entry.getValue();
+
+                        if (pp.getName().toLowerCase(Locale.US).contains("(body)")) {
```

Comment:
Are you able to change `Locale.US` to `Locale.ENGLISH`? This just keeps it consistent with the rest of the codebase

Reply:
@antares1470 It appears in 3 files : FontInfo, GlyphsFrame as well. Shall I change them all?

Reply:
Fixed them all.